source/developer/developer-setup.md

```diff
@@ -33,17 +33,20 @@ Developer Machine Setup
 	4. `git clone https://github.com/<username>/platform.git`  
 	5. `cd platform`
 
-5. Install godep with `go get github.com/tools/godep`
+5. If you are running Mattermost version 3.0 or older, install godep with `go get github.com/tools/godep`
 6. Run unit tests on Mattermost using `make test` to make sure the installation was successful
-7. If the tests passed, you need to create a team and an admin account. You can choose a team name, email and password:
-	- Note: Make sure your team name does not contain any spaces.
-	- `godep go run mattermost.go  -create_team -team_name="name" -email="user@example.com"`
-	- `godep go run mattermost.go  -create_user -team_name="name" -email="user@example.com" -password="mypassword"`
-	- `godep go run mattermost.go  -assign_role -team_name="name" -email="user@example.com" -role="system_admin"`
-
-8. Now you can run Mattermost using `make run`. Log-in as the admin account you created in step 6
-9. You can stop Mattermost using `make stop`
-10. If you want to setup for cross compilation (required for the `make package` and dependant targets) run:
+7. If the tests passed, you can run Mattermost using `make run`
+8. You need to create a team and admin account. You can choose a team name, email and password:
+	- Note: Make sure your team name does not contain any spaces
+    - Note: If you are running Mattermost version 3.0 or older, prepend all commands with `godep `.
+	  E.g. `godep go run mattermost.go  -create_team -team_name="name" -email="user@example.com"`
+	- `go run mattermost.go  -create_team -team_name="name" -email="user@example.com"`
```

Comment:
> When setting up a new test machine, the database contains test accounts
after make test was run.
Due to that, Mattermost will not display the team creation screen on
make run, but rather the login screen.
In order for the developers to log in, they need to create a team and
an admin account first.